samcli/commands/local/lib/local_lambda.py

```diff
@@ -243,33 +247,37 @@ def _make_env_vars(self, function: Function) -> EnvironmentVariables:
         else:
             LOG.debug("No environment variables found for function '%s'", name)
 
-        # This could either be in standard format, or a CloudFormation parameter file format.
+        # This could either be in standard format, or a CloudFormation parameter file format, or mix of both.
         #
         # Standard format is {FunctionName: {key:value}, FunctionName: {key:value}}
         # CloudFormation parameter file is {"Parameters": {key:value}}
+        # Mixed format is {FunctionName: {key:value}, "Parameters": {key:value}}
 
         for env_var_value in self.env_vars_values.values():
             if not isinstance(env_var_value, dict):
-                reason = """
-                            Environment variables must be in either CloudFormation parameter file
-                            format or in {FunctionName: {key:value}} JSON pairs
-                            """
+                reason = "Environment variables {} in incorrect format".format(env_var_value)
                 LOG.debug(reason)
                 raise OverridesNotWellDefinedError(reason)
 
+        overrides = {}
```

Comment:
lets keep overrides to be None if the customer does not provide any env vars.

Reply:
I thought it was redundant to set overrides to None because when overrides is passed to EnvironmentVariables by default it is set to empty dict and lint throws and error when I try to reassign overrides to None if it is empty. Is there a particular reason it should be None?

Reply:
thanks @hnnasit for the clarification. I think it is Ok to keep it empty dict as we already set to empty dict during the env vars processing, so no issue.